samples/azure-quantum/resource-estimation/estimation-cli/README.md

```diff
@@ -0,0 +1,27 @@
+# Resource estimation CLI tool
+
+This sample shows how to build a resource estimation CLI tool that connects to
+Azure Quantum and takes as input a QIR program.
+
+## Setup
+
+The tool requires the `azure-quantum` and `pyqir-generator` libraries in the
+following version.  You can for example use conda to set this up:
```

Comment:
If these are the minimal versions supported and the future versions will work as well, let's clarify this: "of the versions used in the following example or later versions"

Reply:
I cannot guarantee whether future versions will work, so I'd prefer to lock it to these. We can always update the sample for future versions after validating that it still works.